### PR TITLE
feat: documentation and optimizations

### PR DESCRIPTION
**Pull request by JanZachmann**

### CODE REVIEW

File: config/systemd/omnect-ui.service [hunk 1]
```diff
@@ -10,31 +10,31 @@ Type=oneshot
...
-                            -e SSL_KEY_PATH=/cert/device_id_cert_key.pem \
-                        omnectshareddevacr.azurecr.io/omnect-ui:${TAG}"
+ExecStart=/bin/bash -c 'FULL_TAG=$(docker load < /mnt/factory/oci_images/omnect-ui.tar.gz | grep "Loaded image: " | cut -d ':' -f2) ; \                                                                                                        
+                        docker run -d --restart always --name omnect-ui \                                                                                                                                                                     
+                            --user $(id -u omnect_device_socket):$(id -g omnect_device_socket) \                                                                                                                                              
+                            --mount type=bind,source=/run/omnect-device-service/api.sock,target=/socket/api.sock \                                                                                                                            
+                            -v /mnt/cert/priv:/cert \
```
Comment (JoergZeidler) [inline config/systemd/omnect-ui.service:17]: are there TABs added in this file?

File: README.md [hunk 2]
```diff
@@ -5,12 +5,60 @@ This module implements a web frontend and backend to provide omnect specific fea
...
+1.  in all files in config/ folder replace all occurrences of %% *** %% with reasonable values:
+    1.  %%CENTRIFUGO_API_KEY%%: the [API key](https://centrifugal.dev/docs/server/server_api#http-api) should come from a vault
+    2.  %%CENTRIFUGO_TOKEN_HMAC_SECRET_KEY%%: the [HMAC key](https://centrifugal.dev/docs/server/authentication) should come from a vault
+    3.  %%USER%%: user name to be matched on omnect-ui login
+    4.  %PASSWORD%%: password to be matched on omnect-ui login
+2.  it might be appropriate to adapt other default config values to your needs
+3.  inject config files via [omnect-cli](https://github.com/omnect/omnect-cli) into omnect-os image
+```
+# download and copy omnect-ui docker image
+omnect-cli docker inject -d omnectsharedstagingacr.azurecr.io/omnect-portal-omnect-ui:latest -e /oci_images/omnect-ui.tar.gz -i my-omnect-os-image.wic
```
Comment (JoergZeidler) [inline README.md:33]: should we changed the docker registry name to  omnectsharedprodacr?

File: README.md [hunk 3]
```diff
@@ -5,12 +5,60 @@ This module implements a web frontend and backend to provide omnect specific fea
...
+  -k omnect-int-ca-iotedge-staging.key -D 365 \
+  -i my-omnect-os-image.wic
+```
 
 ## Access omnect-ui
 
-browser type: https://hostname:1977
+omnect-ui can be reached at https://DeviceHostnameOrIp:1977<br>
 
+Login with the configured creadentials<br>
```
Comment (JoergZeidler) [inline README.md:58]: typo: credentials

File: config/systemd/omnect-ui.service [hunk 4]
```diff
@@ -10,31 +10,31 @@ Type=oneshot
...
-                            -e CENTRIFUGO_TLS_CERT=/cert/device_id_cert.pem \
-                            -e CENTRIFUGO_TLS_KEY=/cert/device_id_cert_key.pem \
-                            -e CENTRIFUGO_TOKEN_HMAC_SECRET_KEY=${CENTRIFUGO_TOKEN_HMAC_SECRET_KEY} \
-                            -e LOGIN_USER=${LOGIN_USER} \
-                            -e LOGIN_PASSWORD=${LOGIN_PASSWORD} \
-                            -e SOCKET_PATH=/socket/api.sock \
-                            -e SSL_CERT_PATH=/cert/device_id_cert.pem \
-                            -e SSL_KEY_PATH=/cert/device_id_cert_key.pem \
-                        omnectshareddevacr.azurecr.io/omnect-ui:${TAG}"
+ExecStart=/bin/bash -c 'FULL_TAG=$(docker load < /mnt/factory/oci_images/omnect-ui.tar.gz | grep "Loaded image: " | cut -d ':' -f2) ; \
```
[the remark this answers is not in the document]
Comment (JanZachmann) [reply config/systemd/omnect-ui.service:13]: in vsc it is also called fulltag. I'll keep....

**Review by JoergZeidler — APPROVED** [review]

**Review by mlilien — APPROVED** [review]

**Pull request merged**